### PR TITLE
Shippy docs update

### PR DESCRIPTION
**Pull request by dkolas**
This PR replaces references to the ts-cli with instructions for shippy.

Also:
- Update shippy help strings
- Minor silliness

### CODE REVIEW

**Comment by dkolas** [comment]
[SHIP-554]

File: src/steamship/cli/cli.py [hunk 1]
```diff
@@ -48,8 +51,19 @@ def login():
         click.secho(f"🚢🚢🚢 Hooray! You're logged in with user handle: {user.handle} 🚢🚢🚢", fg="green")
 
 
+@click.command()
+def ships():
+    """Ship some ships"""
+    initialize()
+    click.secho("Here are some ships:", fg="cyan")
```
Comment (eob) [inline src/steamship/cli/cli.py:58]: 😂

Comment (eob) [reply src/steamship/cli/cli.py:58]: image

**Review by eob — APPROVED** [review]
LGTM! Thanks a ton for getting the docs too!

**Pull request merged**